EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,81 @@
+## Editorial Style Suggestions
+This document provides editorial style suggestions for the FOCUS v1.0 specifications. It was created in response to a recent discussion about the need for a consistent and professional editorial style to enhance and harmonize FOCUS technical specifications. The guidelines outlined below aim to standardize the presentation of various elements such as **column names**, *column IDs*, [_glossary terms_](), **NORMATIVE KEYWORDS**, **`attribute names`**, *`attribute IDs`* and other key components of the FOCUS documentation.
+
+1. **Column Names**
+   - **Style**: **Bold**, (_with blank space for compound names_)
+   - **Reasoning**: Column names should be easily distinguishable for quick reference and clarity.
+   - **Example**: **Billing Account ID**, **Resource Name**
+
+2. **Column IDs**
+   - **Style**: *Italic*, (_without blank space for compound names, and compound words capatilized_)
+   - **Reasoning**: Italicizing the column IDs helps differentiate them from the display names and other text, making it clear when referring to specific IDs.
+   - **Example**: *BillingAccountId*, *ResourceName*
+
+3. **Glossary Terms**
+   - **Style**: [_Underlined_](), and capitalized
```

Comment:
In GitHub markdown (at least), this renders as underlined, italicized, and a link because markdown doesn't have a native underline. I've seen the <ins>HTML insert tag</ins> used as a workaround before, but not all browsers may chose to make that underlined and not sure what it would look like off hand when we convert to PDF.

The underline used here actually comes from the fact that it's linked, but we had established previously that only the first occurrence of a glossary term in each page should be linked (though this can be revisited). Would the second occurrence also be linked, would it be underlined in another way, or would it be treated as any other text?

Reply:
Based on the discussion I suggest:
## Glossary Usage
* Linked on the first usage of each section (doing by page may prove to be very challenging)
* No italic, no underlined on PDF and HTML printouts
* Capitalized when the term is used in the text.
* 
## Example:
### Section of Standards Text
When configuring the system, ensure that the [Billing Account](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/501#glossary-BillingAccountId) is correctly assigned. The **Billing Account** is crucial for tracking all billing-related activities. Each [Resource]() must adhere to the specified format to avoid conflicts. In case of an error, the **Billing Account** should be verified against the master list. The **Resource** ...

## Glossary:
Definition for:
* **Billing Account**: xxx
* **Resource**: xxx

Comments?